internal/remote/user.go

```diff
@@ -78,6 +78,22 @@ func (user *User) CloseAndFlushOperationQueue() {
 	user.processWG.Wait()
 }
 
+func (user *User) FinishMailboxIDUpdate(tempID string) error {
+	return user.pushOp(&OpRemMailboxTmpID{
+		// Conn metadata is never used, we can se to 0
+		OperationBase: OperationBase{MetadataID: 0},
```

Comment:
If it's not used, can't you just let it default-initialize (integer types default to 0)?